web/src/components/storage/ProposalPageOptions.jsx

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) [2023] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React, { useEffect, useState } from "react";
+import { useHref } from "react-router-dom";
+import { useInstallerClient } from "~/context/installer";
+import { If, ContextualActions } from "~/components/core";
+
+const DASDLink = () => {
+  const href = useHref("/storage/dasd");
+
+  return (
+    <ContextualActions.Item
+      key="dasd-link"
+      href={href}
+      description="Manage and format"
+    >
+      DASD
+    </ContextualActions.Item>
+  );
+};
+
+const ISCSILink = () => {
+  const href = useHref("/storage/iscsi");
+
+  return (
+    <ContextualActions.Item
+      key="iscsi-link"
+      href={href}
+      description="Connect to iSCSI targets"
```

Comment:
With this new approach, it's a lot of easy to add a description for each menu entry and help the users to quickly identify that it's really the action they want to choose. That's why I added this _description_ prop here after talking about that with @ancorgs. Please, check it and propose a better description if you have one in mind. The idea is to avoid using only the acronyms as much as possible too.

Reply:
Just to be sure this leads only to new connections or does it allow also to modify existing/delete connections?

Reply:
Actually, it leads to the iSCSI management page. The idea to use the "Create" word it's based in the fact that probably it's the "most wanted" action

Reply:
In that case I think "Manager connections to iSCSI targets" sounds better to me.